test/fuzz/param.ml

```diff
@@ -1,5 +1,7 @@
 let debug = true
 
-let reference_fuzzing = true
+let reference_fuzzing = false
 
 let initial_fuel = 20
+
+let max_time_execution = 1. (* seconds *)
```

Comment:
That's probably way to much. We should stop around `0.001` by default, otherwise running the 5000 modules is going to take forever.